### DISCUSSION TITLE
Remotion as organization in GSOC 2022 ?

### DISCUSSION THREAD

**Discussion by AbdulBsit** [remotion-dev/remotion#826] [General]
Is there any plan for GSOC 2022 ?
I will love to apply as a contributor

**Comment by tomByrer** [comment]
https://www.quora.com/How-do-I-crack-the-GSOC-2022

**Comment by JonnyBurger** [comment]
The best way is to show interest in solving a particular issue in our issue list, or one of your own if you want us to guide and mentor you  😊 

Always happy to welcome new contributors to and guide them for concrete issues, but we don't have the resources to mentor multiple people over a summer. So we'll not participate unfortunately.
